packaging/ansible-runner-service-project/project/ovirt-host-reconfigure-ovn.yml

```diff
@@ -14,9 +14,18 @@
   when:
     - ansible_facts.packages['rhv-openvswitch'] is defined
 
-- name: Reconfigure OVN for oVirt
-  command: >
-    vdsm-tool ovn-config {{ ovn_central }} {{ ovn_tunneling_interface }} {{ ovn_host_fqdn }}
+- block:
+  - name: Get OVS chassis id
+    command: ovs-vsctl --no-wait get open . external_ids:system-id
+    register: chassis_id
+
+  - name: Reconfigure OVN for oVirt
+    command: vdsm-tool ovn-config {{ ovn_central }} {{ ovn_tunneling_interface }} {{ ovn_host_fqdn }}
+
+  - name: Sync chassis in OVN SB DB
+    command: ovn-sbctl --if-exists chassis-del {{ chassis_id.stdout }}
```

Comment:
It would work, only problem is that this command needs to run as `openvswitch` or `root`, unfortunately delegate_to uses `ovirt` user.

Reply:
Tried it, but it requires some kind of permission file to become the user. I didn't fully understood that, hopefully it will be possible.